aten/src/ATen/native/quantized/AffineQuantizerBase.cpp

```diff
@@ -114,7 +114,7 @@ inline double Round(const double x) {
 }
 #else
 template <class T>
-inline T Round(const T x) {
+static inline T Round(const T x) {
```

Comment:
Is this a false positive or shouldn't the w functions above in the if else also be marked as static

Reply:
Reverted